### ISSUE TITLE
Can't register account

### ISSUE THREAD

**Issue by JonesNMRN** [ctopherngoc/Blossom#108]
Hello, would be great if you make a discord or if i can add you to talk about your project! wanna test it out, can't register it says invalid credits whenever i try.

**Comment by ctopherngoc** [comment]
> Hello, would be great if you make a discord or if i can add you to talk about your project! wanna test it out, can't register it says invalid credits whenever i try.

Seems like the endpoint for my registration link is incorrect. The character creation is also broken due to implementation of new data structure for the characters. I'll address those issues and hopefully have it fixed sometime tomorrow

**Comment by ctopherngoc** [comment]
I believe I fixed all the issues regarding registering, signing in, creating new characters and loading into the game. The new client version V4.0.1 is uploaded in the release section and the new server version is live and running on the VSP server. 

I've set up the discussion board the repro if you want to contact me about this personal project.

If you find another issue you're welcome to open up another ticket.
